target/ppc/translate/vsx-impl.c.inc

```diff
@@ -2197,6 +2197,96 @@ static bool trans_PSTXV(DisasContext *ctx, arg_PSTXV *a)
     return do_lstxv(ctx, &d, true);
 }
 
+static bool do_lstxvp(DisasContext *ctx, arg_D *a, bool store)
```

Comment:
Maybe we could have a single `do_lstxv` that receives a `bool paired` argument.

Reply:
I thought about doing this in the first version, but I opted to keep them simpler and separated, but I'll join them next version